pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -585,7 +585,7 @@ func getLocalIP() ([]v1.NodeAddress, error) {
 							)
 							klog.V(4).Infof("Detected local IP address as %q", ipnet.IP.String())
 						} else {
-							klog.Warningf("Failed to patch IP as MAC address %q does not belong to a VMware platform", vmMACAddr)
+							klog.V(4).Infof("Failed to patch IP as MAC address %q does not belong to a VMware platform", vmMACAddr)
```

Comment:
Can we also log what interface this IP addr belongs to? I think it would help us in the future if we have to debug this again.